stable/polaris/values.yaml

```diff
@@ -104,6 +104,8 @@ webhook:
   objectSelector: {}
   # webhook.rules -- An array of additional for the ValidatingWebhookConfiguration. Each requires a set of apiGroups, apiVersions, operations, resources, and a scope.
   rules: []
+  # webhook.mutatingRules -- An array of additional for the MutatingWebhookConfiguration. Each requires a set of apiGroups, apiVersions, operations, resources, and a scope.
```

Comment:
I think this is missing the word rules, it states `of additional for`. This also looks to be the case for the validating web hook value, which may have been copied for this new value.